plugins/python-build/share/python-build/activepython-2.7.14

```diff
@@ -0,0 +1,13 @@
+case "$(activepython_architecture 2>/dev/null || true)" in
+"linux-x86_64" )
+  install_package "ActivePython-2.7.14.2717-linux-x86_64-glibc-2.12-404899" "http://downloads.activestate.com/ActivePython/releases/2.7.14.2717/ActivePython-2.7.14.2717-linux-x86_64-glibc-2.12-404899.tar.gz" "activepython" verify_py27
```

Comment:
Could you add SHA256 checksum of this archive like other definitions...?

https://github.com/pyenv/pyenv/blob/v1.2.2/plugins/python-build/share/python-build/3.6.4#L5